automatic/notepadplusplus.install/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,30 @@
+﻿$ErrorActionPreference = 'Stop'
+
+$packageName = 'notepadplusplus'
+$url32       = ''
+$url64       = ''
+$checksum32  = ''
+$checksum64  = ''
+
+$packageArgs = @{
+  packageName            = $packageName
+  fileType               = 'EXE'
+  url                    = $url32
+  url64bit               = $url64
+  checksum               = $checksum32
+  checksum64             = $checksum64
+  checksumType           = 'sha256'
+  checksumType64         = 'sha256'
+  silentArgs             = '/S'
+  validExitCodes         = @(0)
+  registryUninstallerKey = 'notepad\+\+'
+}

```

Comment:
There is almost zero need to repeat the urls and checksums, it would make more sense to push them all into the packageArgs instead of above and then just linked.

I'm assuming the regex could handle finding that properly.